data/cube/Cube.js

```diff
@@ -54,6 +54,7 @@ export class Cube {
         info = {},
         lockFn
     }) {
+        makeObservable(this);
```

Comment:
So here is a case where we have a class that doesn't extend `Reactive` using the mobx decorators and activating them with `makeObservable()`. That works a-OK - standard mobx right? - but any sense that we should still extend Reactive as a best practice?